### ISSUE TITLE
cant add fuel?

### ISSUE THREAD

**Issue by firestorm2039** [przemykomo/simple-planes#79]
i have just installed the latest version of this for 1.16.5 and latest version of forge for 1.16.5.

i have made an engine furnace (red thing, i think its called that lol), anyway i placed it in the plane, youtube video said to sit in the plane and right click (i am using ps4 controller), but i right clicked on mouse and nothing happened, another video said press X while holding fuel to add it, this also didnt work.

basically how do i add the fuel to it?
(EDIT): Fuel i have tried, bucket of lava and coal

**Comment by przemykomo** [comment]
Check in controls if there isn't any other keybind at X too. You can change the engine keybind there too.
